guides/core_concepts/0004-pipelines.md

```diff
@@ -0,0 +1,24 @@
+# Pipelines
+
+A Pipeline is a blueprint (therefore a description) made as a chain of [blocks](0003-blocks.html).
+
+A pipeline can be parametric so the same description can be made generic and applied in different
+scenarios that require the same computation steps (but performed with different parameters).
+
+Parameters are then provided when a pipeline is instantiated as a JSON object, and they can be
+accesed using [JSON Path](https://goessner.net/articles/JsonPath/).
+
+Once a pipeline has been instantiated it is called a [Flow](0005-flows.html) and it
+processes [messages](0002-flow-messages.html) (and it takes computation resources as well).
```

Comment:
Flow, and from that moment it starts processing messages, taking up the necessary computation resources